### PR TITLE
Fix appveyor checks

### PR DESCRIPTION
**Pull request by gaborcsardi**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -15,7 +15,15 @@ cache:
...
 # Adapt as necessary starting from here
 
+environment:
+  GITHUB_PAT:
+    secure: Nkvazgyo9FBXlhdY1vdqGsSNJl/DjKXhBYRFd105iEpZxKAXWI5X1aQ/awLgCfWa
+  NOT_CRAN: true
+  USE_RTOOLS: true
+
 build_script:
+  - travis-tool.sh install_github r-lib/devtools
```
Comment (jimhester) [inline appveyor.yml:25]: Why do you need to install devel devtools and testthat?

Comment (gaborcsardi) [reply appveyor.yml:25]: Maybe I don't, I saw failures that looked similar to what I see for `devtools:test()` with CRAN devtools and testthat, but then I was probably wrong.  Let me remove it.

Comment (gaborcsardi) [reply appveyor.yml:25]: We'll see in ~10 minutes: https://ci.appveyor.com/project/gaborcsardi/pkginstall/build/1.0.21

Comment (gaborcsardi) [reply appveyor.yml:25]: Of course it is already in the cache.... anyway, we'll be fine without it now....

**Pull request merged**